src/smart-components/myUserAccess/myUserAccess.js

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect, useState } from 'react';
+import { PageHeader, PageHeaderTitle, Main } from '@redhat-cloud-services/frontend-components';
+import { Text, TextContent } from '@patternfly/react-core';
+import OrgAdminLabel from '../../presentational-components/myUserAccess/orgAdminLabel';
+
+import './myUserAccess.scss';
+import MUAPageSection from '../../presentational-components/myUserAccess/pageSection';
+import MUAOrgEntitlements from '../../presentational-components/myUserAccess/orgEntitlements';
+
+const MyUserAccess = () => {
+
+    const [ isOrgAdmin, setIsOrgAdmin ] = useState();
+    const [ entitlements, setEntitlements ] = useState();
+    const [ isUserReady, setIsUserReady ] = useState(false);
```

Comment:
I would siplify the state management of this component to be object
```JSX
const MyUserAccess = () => {
  const [ user, setUser ] = useState({});

  useEffect(() => {
    insights.chrome.auth.getUser().then(({ identity, entitlements }) => (
      setUser({ entitlements, isOrgAdmin: identity?.user?.is_org_admin })
    ));
  }, [])
  return (
    <React.Fragment>
        { user.hasOwnProperty(entitlements) && user.hasOwnProperty(isOrgAdmin) ?
          <React.Fragment>
             {/* ... */}
          </React.Fragment> : <span> test </span>
        }
    </React.Fragment>      
  );
}
```

Also the `<span> test </span>` should be replaces with spinner, right?